crates/lib3h/src/transport/protocol.rs

```diff
@@ -50,6 +52,78 @@ pub type DynTransportActor = Box<
     >,
 >;
 
+/// this is awkward... I *think* it'll go away with ghost-actor proc-macro
+pub struct TransportEndpointAsActor {
```

Comment:
About to say it was awkward and hope you're right about the macro eliminating it.